delta/src/main/scala/io/qbeast/spark/delta/DeltaMetadataManager.scala

```diff
@@ -28,6 +28,8 @@ import org.apache.spark.sql.SparkSession
  */
 object DeltaMetadataManager extends MetadataManager {
 
+  private val deltaLogCache = scala.collection.mutable.Map[QTableID, DeltaLog]()
```

Comment:
Could this structure affect the memory? If so, how much? (I guess the worst case scenario is to manage a big Catalog of tables in the same Spark Session...)

Reply:
I removed the `deltaLogCache` as actually it was not producing the overhead